### PR TITLE
doc improvement: style fixes in nRF9160 samples part-2

### PR DESCRIPTION
**Pull request by richabp**
Fixed inconsistencies in style and structure

Ref: NCSDK-13404

Signed-off-by: Richa Pandey <richa.pandey@nordicsemi.no>

### CODE REVIEW

File: samples/nrf9160/http_update/application_update/README.rst [hunk 1]
```diff
@@ -94,7 +94,7 @@ Remember to do the following when specifying the filenames:
 * Use the ``<bucket-name>.s3.<region>.amazonaws.com`` part of the URL as the hostname of the server hosting the images, without including ``https://``.
 * Specify the file names as the remaining part of the URL.
 
-You can then use the steps mentioned in the `Specifying the image files`_ section above to upload both images.
+Use the steps mentioned in the `Specifying the image files`_ section to upload both images.
```
Comment (peknis) [inline samples/nrf9160/http_update/application_update/README.rst:97]: Not sure about using steps. Maybe follow the steps?

Comment (richabp) [reply samples/nrf9160/http_update/application_update/README.rst:97]: Yes, that sounds better. Thanks!

File: samples/nrf9160/lte_ble_gateway/README.rst [hunk 2]
```diff
@@ -37,6 +25,18 @@ The sample also requires a `Nordic Thingy:52`_.
 
 .. include:: /includes/spm.txt
 
+Overview
+*********
+
+The sample connects using Bluetooth LE to a Thingy:52 running the factory pre-loaded application.
```
Comment (peknis) [inline samples/nrf9160/lte_ble_gateway/README.rst:31]:
```suggestion
The sample connects using Bluetooth LE to a Thingy:52 running the factory preloaded application.
```
https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules#Terms,abbreviations,andspellingrules-P

File: samples/nrf9160/sms/README.rst [hunk 3]
```diff
@@ -28,6 +18,14 @@ The sample supports the following development kit:
    :header: heading
    :rows: nrf9160dk_nrf9160_ns
 
+Overview
+********
+
+The SMS sample registers the nRF9160-based device for SMS service within the nRF9160 modem using the :ref:`sms_readme` library.
+The sample requires an LTE connection.
+
+When the sample starts, it sends SMS if a recipient phone number is set in the configuration.
```
Comment (peknis) [inline samples/nrf9160/sms/README.rst:27]:
```suggestion
When the sample starts, it sends an SMS if a recipient phone number is set in the configuration.
```

**Review by peknis — CHANGES_REQUESTED** [review]

File: samples/nrf9160/http_update/full_modem_update/README.rst [hunk 4]
```diff
@@ -10,6 +10,21 @@ nRF9160: HTTP full modem update
...
+The sample supports the following development kit, version 0.14.0 or higher:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns
+
+.. include:: /includes/spm.txt
+
+On the nRF9160 DK, you must set the control signal from the nRF52840 board controller MCU (**P0.19**) to *high* to let the nRF9160 communicate with the external flash memory.
+To do so, enable the ``external_flash_pins_routing`` node in devicetree.
```
[the remark this answers is not in the document]
Comment (richabp) [reply samples/nrf9160/http_update/full_modem_update/README.rst]: Done.

File: samples/nrf9160/http_update/modem_delta_update/README.rst [hunk 5]
```diff
@@ -30,6 +23,12 @@ The sample supports the following development kit:
 
 .. include:: /includes/spm.txt
 
+Overview
+********
+
+The sample connects to an HTTP server to download a signed modem delta image.
+The delta images are part of the official modem firmware releases.
+The sample will automatically download the correct delta image to go from the base version to the test version, or the test version to the base version, depending on the currently installed version.
```
[the remark this answers is not in the document]
Comment (richabp) [reply samples/nrf9160/http_update/modem_delta_update/README.rst]: Done.

File: samples/nrf9160/lte_ble_gateway/README.rst (hunk 2 above)
Comment (richabp) [reply samples/nrf9160/lte_ble_gateway/README.rst]: Done.

File: samples/nrf9160/sms/README.rst (hunk 3 above)
Comment (richabp) [reply samples/nrf9160/sms/README.rst]: Done.

**Review by peknis — APPROVED** [review]

**Review by VeijoPesonen — APPROVED** [review]
Looks good to me when it comes to LwM2M client sample.

**Pull request merged**